nestkernel/connector_model_impl.h

```diff
@@ -558,32 +559,50 @@ GenericConnectorModel< ConnectionT >::delete_connection( Node& tgt,
 // by modules                                                                  //
 /////////////////////////////////////////////////////////////////////////////////
 
+inline bool
+ends_with( std::string const& value, std::string const& ending )
+{
+  if ( ending.size() > value.size() )
+    return false;
+  return std::equal( ending.rbegin(), ending.rend(), value.rbegin() );
+}
```

Comment:
This function could be useful elsewhere. Should it move to a `string_utils.h` or similar in `libnestutil`?